### PR TITLE
Add PagingSample

### PR DESCRIPTION
**Pull request by ChrisCraik**
Adds a PagingSample for the newly released Paging library

### CODE REVIEW

File: PagingSample/README.md [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+* [LiveData](https://developer.android.com/reference/android/arch/lifecycle/LiveData.html)
+
+## Features
+
+This sample contains a single screen with a list of text items. Items can be added to the list with the input at the top, and swiping items in the list removes them.
+
+## License
+-------
+
+Copyright 2015 The Android Open Source Project, Inc.
```
Comment (yigit) [inline PagingSample/README.md]: 2017

File: PagingSample/app/build.gradle [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+            minifyEnabled false
+            proguardFiles getDefaultProguardFile('proguard-android.txt'), 'proguard-rules.pro'
+        }
+    }
+}
+
+dependencies {
+    implementation fileTree(dir: 'libs', include: ['*.jar'])
+    implementation "com.android.support:appcompat-v7:${versions.support}"
+    implementation "com.android.support.constraint:constraint-layout:${versions.constraint_layout}"
```
Comment (yigit) [inline PagingSample/app/build.gradle]: actually i think we can remove this.

Comment (ChrisCraik) [reply PagingSample/app/build.gradle]: Done

File: PagingSample/README.md (hunk 1 above)
Comment (ChrisCraik) [reply PagingSample/README.md]: Done

**Pull request merged**